src/main/java/com/kaziamyr/onlinebookstore/dto/UserLoginRequestDto.java

```diff
@@ -0,0 +1,17 @@
+package com.kaziamyr.onlinebookstore.dto;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+
+@Data
+public class UserLoginRequestDto {
+    @NotNull
+    @Email
+    @Size(min = 1, max = 100)
```

Comment:
Is there are any requirements that password and email should be 100 symbols maximum?